trampoline_v2.sh

```diff
@@ -285,6 +285,11 @@ do
     fi
 done
 
+# We want to support legacy style TRAMPOLINE_BUILD_FILE used with V1
+# script: e.g. "github/repo-name/.kokoro/run_tests.sh"
+TRAMPOLINE_BUILD_FILE="${TRAMPOLINE_BUILD_FILE#github/[^/]*/}"
```

Comment:
Might want to do `+` instead of `*`, but it probably doesn't matter.

Reply:
done